docs/semgrep-supply-chain/triage-remediation.md

```diff
@@ -89,7 +89,7 @@ Use filters to narrow down your results. The following criteria are available fo
 | **Branch**             | Filter by findings in different Git branches. |
 | **Teams**              | Filter for findings in projects to which the specified teams are associated with. Available only to organizations with [Teams](/deployment/teams#teams-beta) enabled. |
 | **Tags**               | Filter for findings based on the tags associated with the project. |
-| **Status**             | Filter the triage state of a finding: <ul><li>**New**: Vulnerabilities that have not undertaken triage or remediation action.</li><li>**Fixed**: Vulnerabilities that are no longer detected after a scan. This typically means that the dependency containing the vulnerability has been updated. Semgrep Supply Chain automatically checks if the dependency has been updated and sets the vulnerability's status as Fixed.</li><li>**Ignored**: Vulnerabilities that have been triaged as ignored by the user. </li></ul>|
+| **Status**             | Filter the triage state of a finding: <ul><li>**Open**: Findings for which there have been no triage or remediation action.</li><li>**Reviewing**: Findings that require more investigation to determine what the next steps should be.</li><li>**Fixing**: Findings for which a Jira ticket has been created.</li><li>**Fixed**: Vulnerabilities that are no longer detected after a scan. This typically means that the dependency containing the vulnerability has been updated. Semgrep Supply Chain automatically checks if the dependency has been updated and sets the vulnerability's status as **Fixed**.</li><li>**Ignored**: Vulnerabilities that have been triaged as **Ignored** by the user. </li></ul>|
```

Comment:
@khorne3 I checked and both Fixing and Reviewing can make Jira tickets, so maybe Fixing should be updated, as it is not exclusively the only triage state for which a jira ticket can be made?

Reply:
It looks like Open, Fixing, and Reviewing allow for the creation of tickets, so wdyt about leaving as-is, since that's the status indicating that a ticket has been created (which is a bit different from having ticket creation being tied to a specific status)?

Reply:
How is **Fixing** "the status indicating that a ticket has been created"? I mean, I just tried to set something to Fixing w/o a Jira ticket and it allowed me to. Shouldn't people also make a jira ticket if something needs more investigation (review)? Marrying the concept of "Fixing" = jira ticket seems odd considering that there's no strict enforcement of ticket creation.

Reply:
Oh, I see what you're saying -- my understanding is that the fixing status indicates that someone has created a ticket (and yes, there's no enforcement, it's just a marker that the task of creating a ticket has been done so that people can track what finding has been looked at). I guess someone could do the same w/ reviewing, but it doesn't seem like that's how we defined it? This seems like product decision so I'll let @jsmoxon chime in on how he wants us to handle these defintions

Reply:
We changed this later when we allowed issue ticket creation for more states. I think now I'd writing Fixing as:
"Findings for which you have decided a fix is needed"
Then, could optionally add something like about this being the most common state for issues tracked in Jira or assigned to a developer, etc.